### ISSUE TITLE
when runing demo_hidden.py

### ISSUE THREAD

**Issue by Nortinwell** [bryanyzhu/Hidden-Two-Stream#29]
Hi,
   When I run demo_hidden.py following your tutorial, it will have the following error:
`I0323 16:50:23.343171 19562 net.cpp:753] Ignoring source layer action_loss
we got 3783 test videos
/home/zh/Hidden-Two-Stream-master/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c01
(0, 13)
/home/zh/Hidden-Two-Stream-master/UCF-101/ApplyEyeMakeup/v_ApplyEyeMakeup_g01_c02
(0, 0)
Killed`
I don't know why it kills the process (my card is GTX 1080ti, and i changed the batch_size to 10)

**Comment by Nortinwell** [comment]
When i training and finetuning from ucf101_split1_vgg16_init.caffemodel, my loss = nan at begin. I haven't changed the other parameters. Why does this happen?

**Comment by bryanyzhu** [comment]
Because the MotionNet is also fine-tuning, the network is easy to diverge in the beginning. Just try more runs, if it doesn't diverge in the first 100 iterations, it will be good.

If you continuously get NaN after multiple runs, you can lower your learning rate for the first 500 iterations. And then resume the training with a higher learning rate. That should solve the problem too.

**Comment by Nortinwell** [comment]
Thank you for your advice. I used batch_size 1 for training later, but I found the accuracy rate was only 0.01 after the end. Did I ignore any important steps?

**Comment by bryanyzhu** [comment]
Batch size could also impact the performance, try use a larger batch size. If you use exactly the same recipe as I do, you should get accuracy of 30+ after the first epoch, and accuracy of 70+ when the training is finished.
